thinc/backends/ops.py

```diff
@@ -878,39 +878,41 @@ def backprop_gelu(
             return dY
         return dY * dX
 
-    def mish(self, X: Floats2d, threshold: float = 20.0) -> Floats2d:
-        Y = self.alloc2f(*X.shape, dtype=X.dtype)
+    def mish(
+        self, X: FloatsType, threshold: float = 20.0, inplace: bool = False
+    ) -> FloatsType:
         tmp = X * self.xp.tanh(self.xp.log(1.0 + self.xp.exp(X)))
```

Comment:
I think so. But that seems to be a nice addition for a separate PR (do people use softplus much?)?

Reply:
I haven't seen it much other than in cases where one is predicting something that has to be positive e,g: https://github.com/geoopt/geoopt/blob/master/geoopt/manifolds/stereographic/manifold.py#L526. Here the distance predictor has it https://arxiv.org/pdf/2110.04126.pdf as well. But I don't think its very common otherwise.